Tests/BasicPerformanceTests/OutputByteStreamPerfTests.swift

```diff
@@ -190,6 +190,32 @@ class OutputByteStreamPerfTests: XCTestCase {
             }
         }
     }
+
+    func testJSONToString_X100() {
+        let foo = JSON.dictionary([
+            "foo": .string("bar"),
+            "bar": .int(2),
+            "baz": .array([1, 2, 3].map(JSON.int)),
+            ])
+
+        let bar = JSON.dictionary([
+            "poo": .array([foo, foo, foo]),
+            "foo": .int(1),
+            ])
+
+        let baz = JSON.dictionary([
+            "poo": .array([foo, bar, foo]),
+            "foo": .int(1),
+            ])
```

Comment:
It's probably worth using larger test inputs, because the cases we care about are where we write large JSON strings, and this will otherwise end up oversampling the JSON -> String setup overhead (cost of constructing any String, regardless of size).